hack/xref-helpmsgs-manpages

```diff
@@ -216,9 +216,12 @@ sub xref_by_help {
                 # autocompletion that looks like a Go template, but those
                 # template options aren't documented in the man pages.
                 if ($k eq '--format' && ! ref($man->{$k})) {
-                    warn "$ME: 'podman @subcommand': --format options are available through autocomplete, but are not documented in $man->{_path}\n";
-
-                    ++$Errs   unless "@subcommand" eq "inspect";
+                    # "podman inspect" tries to autodetect if it's being run
+                    # on an image or container. It cannot sanely be documented.
```

Comment:
This warning has been present for a few months. I left it in because maybe I could find a way to document `podman inspect --format`? But nope, I don't think it's practical. Here I simply remove the warning.